****************************************************************
*                      SLURM Batch System                      *
*           IN2P3 Computing Centre, Villeurbanne FR            *
****************************************************************
* Date:                 Tue Apr 25 16:15:32 CEST 2023
* Job name:             run_py_subp.sh
* Job id:               27318382
* User:                 namourou
* Account:              lsst
* Submit host:          cca004
* Partition:            htc
* Quality of service:   normal
* Nodelist:             ccwslurm0328
****************************************************************
DESC setup
GO1
Incorporation of magnitudes in AMICO catalog

Catalog loaded at /sps/lsst/groups/clusters/amico_validation_project/catalogs/AMICO/raw_amico_cats/cosmoDC2_small_photoz_flexzboost_v1_yband/DETECTIONS_DERIVED/ and will be saved in /sps/lsst/groups/clusters/amico_validation_project/catalogs/AMICO/amico_cats/amico_map_associations_flxzb_mag/mag_y/

processing 10070
Healpix list : [10070, 10069, 10071, 10197, 10198, 10325, 9814, 9941, 9942] 
Now merging all input healpix files
Done. 
Incorporation of magnitudes
57394 0.1
117809 0.2
238637 0.4
359466 0.6
480295 0.8
601124 1.0
  GALID    FIELD_PROB ASSOC_ID ...       mag_y        mt_id_final
---------- ---------- -------- ... ------------------ -----------
7938499358  0.9750211     1737 ... 24.363115126162626     9971304
7938506058 0.97086084     1737 ...  24.84926782478385     9977647
7938659819 0.97182125     1737 ... 24.902884992824685    10029472
7938689334  0.9029641     1737 ...  24.33206328587622    10057407
7938770500  0.9624931     1737 ...  24.85718695030825    10076129
There are : 0 galaxies which are not found inside input catalog
Program ran in : 456.50093960762024
Done.
G02
Reading amico files before merging
Configuration arguments:  10070
Traceback (most recent call last):
  File "/pbs/home/n/namourou/test_jupyter/cluster_challenge/prepare_catalogs/read_amico_mb_subp.py", line 33, in <module>
    table = Table.read(cat_path)
NameError: name 'cat_path' is not defined. Did you mean: 'outpath'?
GO3
****************************************************************
*                      SLURM Batch System                      *
*           IN2P3 Computing Centre, Villeurbanne FR            *
****************************************************************
Date: Tue Apr 25 16:23:19 CEST 2023
Job informations can be found using these commands:
Accounting:
sacct -j 27318382
Efficiency:
seff 27318382
****************************************************************
